data/human/weapons.txt

```diff
@@ -787,30 +781,43 @@ outfit "Meteor Missile Pod"
 		sound "meteor"
 		ammo "Meteor Missile"
 		icon "icon/meteor"
+		"submunition" "Meteor Homing"
 		"fire effect" "meteor fire"
-		"die effect" "missile death"
 		"hit effect" "missile hit"
-		"inaccuracy" 5
+		"inaccuracy" 2
 		"velocity" 0
 		"velocity override" 20
-		"lifetime" 192
+		"lifetime" 56
 		"range override" 3850
 		"reload" 200
 		"burst count" 7
 		"burst reload" 80
 		"firing energy" 1
 		"firing heat" 20
-		"acceleration" .4
-		"drag" .02
-		"turn" 2
-		"homing" 1
+		"acceleration" .5
+		"drag" .025
+		"missile strength" 10
+	description "The Meteor Missile is the cheapest and simplest homing weapon available in human space. Meteors have limited maneuvering capabilities, and their infrared tracking systems only activate after a second of flight. However, Meteor Missiles are able to reach high velocities, leaving targets with little time to react. These qualities make Meteor Missiles effective at a distance, but they will struggle to lock onto nearby enemies."
+	description "	Pods enable fighters and other light craft to carry small quantities of missiles into battle while retaining enough space for other weapons. Large ships are better off with full-size launchers."
+
+outfit "Meteor Homing"
+	weapon
+		sprite "projectile/meteor"
+			"no repeat"
+			"frame rate" .25
+		"die effect" "missile death"
+		"hit effect" "missile hit"
+		"velocity" 0
+		"lifetime" 136
+		"acceleration" 1.6
+		"drag" .08
+		"turn" 1.1
+		"homing" 4
```

Comment:
In testing this with the 17 gun port Bun ship, I noticed something strange with this homing behavior. Even when pointed directly at the target many of the the missiles veer off course  in this transition, presumably from having a bad lock, despite previously moving directly towards the target. This ends up meaning that what would be a good hit with no homing -becomes- a miss when the homing activates.

Probably not super noticeable during standard play between AIs, but it's almost surely gonna get us bug reports from edge cases in player use.

Reply:
That's standard missile haywire behaviour. (#4212)

Reply:
Right, but it kicking in because of the transition from dead fire to homing -looks- like non/standard behavior.

Not really any way to fix it, just pointing it out as something that will probably crop up :)